### PR TITLE
Update README and add Filter Scripts

### PR DESCRIPTION
**Pull request by bannanc**
I added filter_molecules.py the script I used to filter DrugBank and eMolecules and updated the README.md document, including some changes to the formatting.

### CODE REVIEW

**Comment by bannanc** [comment]
There is some redundancy in the filter code because I handled DrugBank and eMolecules differently, but I generalized the methods for eMolecules enough that the script is re-useable if we wanted to filter a different large database.

**Comment by bannanc** [comment]
@vtlim 
It is better practice that I don't merge my own request so if you could take a quick look and merge this that would be great.

File: README.md [hunk 1]
```diff
@@ -1,28 +1,38 @@
...
+**NOTE: only Python 2 is supported for the time being.**
 
-NOTE: only Python 2 is supported for the time being.
+### Project Goals
 
+This project relates to the larger goals of the 
+[Open Force Field Group's](https://github.com/open-forcefield-group)
+effort to automate force field parameterization. 
+
+Here we have created scripts to minize small molecules using a variety of force fields with two goals in mind.
```
Comment (vtlim) [inline README.md]: "minimize"

File: README.md [hunk 2]
```diff
@@ -1,28 +1,38 @@
...
-NOTE: only Python 2 is supported for the time being.
+### Project Goals
 
+This project relates to the larger goals of the 
+[Open Force Field Group's](https://github.com/open-forcefield-group)
+effort to automate force field parameterization. 
+
+Here we have created scripts to minize small molecules using a variety of force fields with two goals in mind. 
+1. Find places where the SMIRNOFF99Frosst force field behaves differently from currently accepted force fields.
+2. Discover molecules that are minized to different conformations by different force fields. These molecules will likely be used in future SMIRNOFF parameterizations.
```
Comment (vtlim) [inline README.md]: "minimized"

File: README.md [hunk 3]
```diff
@@ -1,28 +1,38 @@
...
+* OPLS3
+* OPLS2005
 
 ### Contents
 
 * *ffcompare.py*: Script to read in mol2 files and run minimization using OpenMM (or oechem.OESzybki).
 
 * *genTriposGAFFandGAFF2.py*: Script to generate mol2 files with Tripos, GAFF, and GAFF2 atom types.
 
-TODO: remove anything related to 'commands' module.
```
Comment (vtlim) [inline README.md]: Did we move this `TODO` somewhere else? We don't want to delete / forget it I presume...

**Comment by vtlim** [comment]
@bannanc, do you also include an example command to use each of the filter scripts somewhere (or did I miss it in the README or something....)?

File: README.md (hunk 3 above)
Comment (vtlim) [inline README.md]: never mind, I found the related github issue.

**Comment by bannanc** [comment]
@vtlim I fixed the typos you pointed out above, I'm going to suggest that if we have suggested changes then we don't merge a pull request. I'll open a new one with the README changes.

**Pull request merged**